src/components/Banners.tsx

```diff
@@ -1,15 +1,29 @@
 import { FC } from 'react';
+import { useRouter } from 'next/router';
+import { Box, Link } from '@chakra-ui/react';
+
+import { Banner } from './UI';
+
+import { ACADEMIC_GRANTS_2023_URL } from '../constants';
 
 interface Props {}
 
 export const Banners: FC<Props> = () => {
-  // LEAVING THESE COMMENTS AS A REFERENCE
-
-  // const router = useRouter();
+  const router = useRouter();
 
-  // if (!router.pathname.includes(URL)) {
-  // add application banner here
-  // }
+  if (!router.pathname.includes(ACADEMIC_GRANTS_2023_URL)) {
+    return (
+      <Banner>
+        <Box fontSize='paragraph' textAlign='center'>
+          Applications are open for the{' '}
+          <Link fontWeight={700} href={ACADEMIC_GRANTS_2023_URL}>
+            Academic grants
```

Comment:
```suggestion
            Academic Grants Round
```

I think it should be this since on the pages that's how its referred to.

Reply:
<img width="1427" alt="Screen Shot 2023-01-16 at 11 36 46 AM" src="https://user-images.githubusercontent.com/15589226/212746323-62a3d0f2-5709-406a-b5c7-b8353db8d1ea.png">

Reply:
Agreed